src/generated/resources/assets/techarium/lang/en_us.json

```diff
@@ -1,13 +1,19 @@
 {
+  "block.techarium.aluminium_ore": "Aluminium Ore",
```

Comment:
Did you add these manually...? These should be generated in the LangDataGenerator

Reply:
used datagen but seems to have gotten wiped when i rebased